R/JHU.R

```diff
@@ -0,0 +1,125 @@
+#' Filter JHU data for target country
+#' @description Downloads daily covid data from JHU (via get_JHU_data)
+#' and subsets it for a given target country
+#' @param target_country A string with the country of interest
+#' @return A data frame of COVID cases for the target country
+#' @importFrom dplyr %>% filter select
+#' @importFrom rlang .data
+#' 
+check_alternate_data_source <- function(target_country){
+  warning(paste0("'", target_country, "' has no direct data source: using JHU for data"))
+  # make first letter uppercase so it can find resource
+  target_country_ <- paste0(toupper(substr(target_country, 1, 1)),
+                          substr(target_country, 2, nchar(target_country)))
+  
+  # get the full data set
+  data <- get_JHU_data()
+  
+  # check country is in the data
+  if (!(target_country_ %in% unique(data$country))){
+    stop(paste0("Country '", target_country, "' has no data."))
+  }
+  
+  # filter data for requested country
+  data <- data %>% dplyr::filter(.data$country==target_country_)
```

Comment:
condense these lines. Think comment = paragraph aim and code = words of paragraph

Reply:
done